lib/manager/bundler/locked-version.ts

```diff
@@ -0,0 +1,24 @@
+const DEP_REGEX = '(?<=\\().*(?=\\))';
+export function extractLockFileEntries(lockFileContent: string): any {
```

Comment:
```suggestion
export function extractLockFileEntries(lockFileContent: string): Map<string,string> {
```
Please declare a type instead of `any`